sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -78,11 +78,8 @@ static bool cpu_capture_is_locked(mmio_region_t base_addr) {
  */
 static bool rstmgr_software_reset_is_locked(
     mmio_region_t base_addr, dif_rstmgr_peripheral_t peripheral) {
-  uint32_t bitfield =
-      mmio_region_read32(base_addr, RSTMGR_SW_RST_REGWEN_REG_OFFSET);
-
-  // When bit is cleared, the software reset for `peripheral` is disabled.
-  return !bitfield_bit32_read(bitfield, peripheral);
+  return !mmio_region_read32(
+      base_addr, RSTMGR_SW_RST_REGWEN_0_REG_OFFSET + 4 * peripheral);
```

Comment:
We prefer an explicit `sizeof(uint32_t)` to make the `4` value more meaningful.

Reply:
yeah good point.